cmd/ddev/cmd/z_rm_test.go

```diff
@@ -27,7 +27,6 @@ func TestDevRm(t *testing.T) {
 		assert.Contains(string(out), format("Stopping %s-db ... done", app.ContainerName()))
 		assert.Contains(string(out), format("Removing %s-web ... done", app.ContainerName()))
 		assert.Contains(string(out), format("Removing %s-db ... done", app.ContainerName()))
-
-		cleanup()
```

Comment:
cleanup was perhaps poorly named. It doesn't remove the site, it just reverts back to the original working directory, which feels polite?